java/ql/lib/ext/java.util.model.yml

```diff
@@ -360,21 +362,34 @@ extensions:
       pack: codeql/java-all
       extensible: neutralModel
     data:
-      - ["java.util", "Collections", "emptyList", "()", "manual"]
+      - ["java.util", "ArrayList", "ArrayList", "(int)", "manual"]
+      - ["java.util", "ArrayList", "size", "()", "manual"]
+      - ["java.util", "Collection", "isEmpty", "()", "manual"]
       - ["java.util", "Collection", "size", "()", "manual"]
+      - ["java.util", "Collections", "emptyList", "()", "manual"]
+      - ["java.util", "Collections", "emptyMap", "()", "manual"]
+      - ["java.util", "Collections", "emptySet", "()", "manual"]
       - ["java.util", "Iterator", "hasNext", "()", "manual"]
+      - ["java.util", "List", "clear", "()", "manual"]
       - ["java.util", "List", "contains", "(Object)", "manual"]
       - ["java.util", "List", "isEmpty", "()", "manual"]
       - ["java.util", "List", "size", "()", "manual"]
+      - ["java.util", "Map", "clear", "()", "manual"]
```

Comment:
Most likely something similar as for List.clear.

Reply:
Updated both in https://github.com/github/codeql/pull/11779/commits/fb6725ddaa4a62466b23046c36a686cee7e00da4.